pkg/command/plugin.go

```diff
@@ -259,16 +266,23 @@ func newUpgradePluginCmd() *cobra.Command {
 				return errors.New("invalid target specified. Please specify correct value of `--target` or `-t` flag from 'kubernetes/k8s/mission-control/tmc'")
 			}
 
-			pluginVersion, err := pluginmanager.GetRecommendedVersionOfPlugin(pluginName, getTarget())
-			if err != nil {
-				return err
+			var pluginVersion string
+			if config.IsFeatureActivated(constants.FeatureCentralRepository) {
+				// With the Central Repository feature we can simply request to install
+				// the recommendedVersion.
+				pluginVersion = cli.VersionLatest
+			} else {
+				pluginVersion, err = pluginmanager.GetRecommendedVersionOfPlugin(pluginName, getTarget())
+				if err != nil {
+					return err
+				}
 			}
 
 			err = pluginmanager.UpgradePlugin(pluginName, pluginVersion, getTarget())
 			if err != nil {
 				return err
 			}
-			log.Successf("successfully upgraded plugin '%s' to version '%s'", pluginName, pluginVersion)
```

Comment:
The "version" info is removed from this printout because with the more efficient SQL DB, we don't know which version we are upgrading too here, we just as to install the `recommendedVersion`, whatever that version is.

This change in printout is not a big deal because it is preceded by another printout mentioning the version:
```
ℹ  Installing plugin 'management-cluster:v0.0.1' with target 'mission-control'
✔  successfully upgraded plugin 'management-cluster'
```